system-tests/projects/v8-snapshot/require-full-path-var/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "require-full-path-var"
+}
```

Comment:
It could be helpful to include README's in these system test packages you added to describe what specific scenarios are being tested. I can glean some context from the names, but more might be needed as time goes on.

Reply:
Added here: [`3accc37` (#23566)](https://github.com/cypress-io/cypress/pull/23566/commits/3accc371181a876d646b3c3cf5c35dd60440be3b)